strpipe/ops/tests/test_pad.py

```diff
@@ -45,7 +47,8 @@ def test_pad_passing_state_needs_both_sos_and_eos_or_neither():
     ]
 
     state1 = {'maxlen': 0, 'pad_token': '<PAD>'}
-    state2 = {'maxlen': 2, 'pad_token': '<PAD>', 'sos_token': '<sos>', 'eos_token': '<eos>'}
+    state2 = {'maxlen': 2, 'pad_token': '<PAD>',
+              'sos_token': '<sos>', 'eos_token': '<eos>'}
```

Comment:
Strange coding style. Either one value per line or all in single line

Reply:
4c85da4

Reply:
f210f41

Reply:
ea355b3